misc/userscripts/add-nextcloud-bookmarks

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python
+
+"""
+Behavior:
+    A qutebrowser userscript that creates bookmarks in Nextcloud's Bookmarks app.
+
+Requirements:
+    requests
+
+userscript setup:
+    Optionally create ~/.config/qutebrowser/add-nextcloud-bookmarks.ini like:
+
+[nextcloud]
+HOST=https://nextcloud.example.com
+USER=username
+;PASSWORD=lamepassword
+DESCRIPTION=None
+;TAGS=just-one
+TAGS=read-me-later, added-by-qutebrowser
```

Comment:
Any reason no to support comma separated with no spaces? That seems more typical to me. If you want to support both just call `.strip()` on each of the tags when parsing it